src/options.jl

```diff
@@ -36,3 +37,10 @@ function needs_alignment(opts::Options)
         opts.align_pair_arrow ||
         opts.align_matrix
 end
+
+valid_for_in_op(s::String) = s in ("in", "=", "∈")
+
+function validate_options(opts::Options)
+    @assert valid_for_in_op(opts.for_in_replacement)
+    opts
+end
```

Comment:
@domluna btw I didn't find a good "central" place to validate the options (except perhaps for the place where styles are expanded, but that's very raw), so hooking this into the `State` constructor seemed like the best way. Please lmk in case there's a better place.